x-pack/plugins/observability/public/plugin.ts

```diff
@@ -52,9 +53,11 @@ export interface ObservabilityPublicPluginsSetup {
   data: DataPublicPluginSetup;
   triggersActionsUi: TriggersAndActionsUIPublicPluginSetup;
   home?: HomePublicPluginSetup;
+  usageCollection: UsageCollectionSetup;
```

Comment:
same as my previous comment: `kibana.json` declares this dependency optional. We should account for it in TS.